internal/host/statemachine.go

```diff
@@ -549,6 +549,22 @@ func NewHostStateMachine(sm stateswitch.StateMachine, th TransitionHandler) stat
 		},
 	})
 
+	sm.AddTransitionRule(stateswitch.TransitionRule{
+		TransitionType: TransitionTypeRefresh,
+		SourceStates: []stateswitch.State{
+			stateswitch.State(models.HostStatusPreparingForInstallation),
+			stateswitch.State(models.HostStatusPreparingFailed),
```

Comment:
Why not only `HostStatusPreparingForInstallation` ?

Reply:
Good point, will change this.